test/Algebra/Graph/Test/Generic.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE GADTs, RankNTypes, ViewPatterns #-}
+{-# LANGUAGE GADTs, NegativeLiterals, RankNTypes, ViewPatterns #-}
```

Comment:
I don't think you are actually using the `NegativeLiterals` extension.

If you'd like to use it, you'll need to switch from `show % (vertex (-1))` to `show % (-1)`. 

But if you prefer to leave all tests as they are, then I guess we should drop `NegativeLiterals` from this list.

Reply:
Right! I used to make use of that extension at an earlier point but your suggested tests removed the need. Fixed now!